README.md

```diff
@@ -52,8 +52,8 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Tuts+](https://code.tutsplus.com/articles/call-for-authors-write-for-tuts--cms-22034) - $100 (Quick tip) $250 (Tutorial)
   > Technical focused articles. Pick from a list of possible articles.
 
-- [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Applications are currently closed.
-  > Technical focused articles. Specific on Flutter.
+- [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Pay unknown
+  > Technical articles about Flutter.
```

Comment:
Apps appear to be open again, but pay is unknown.

Reply:
Hello, they recruit writers who have posted articles on Medium that talk about codeMagic.

Reply:
Ok, do you know how much they pay? I can add more details if you have any.